ibm_catalog.json

```diff
@@ -197,10 +205,22 @@
               ]
             },
             {
-              "key": "blacklisted_ports"
+              "key": "blacklisted_ports",
+              "type": "array",
```

Comment:
this should not be a textarea. Thats only needed for list(object) types. This is `list(number)` type